src/test/java/uk/gov/hmcts/reform/finrem/caseorchestration/config/NotificationServiceConfigurationTest.java

```diff
@@ -19,6 +19,7 @@ public void shouldReturnTheConfiguration() {
         assertEquals("/consent-order-made", underTest.getConsentOrderMade());
         assertEquals("/consent-order-not-approved", underTest.getConsentOrderNotApproved());
         assertEquals("/consent-order-available", underTest.getConsentOrderAvailable());
+        assertEquals("/prepare-for-hearing", underTest.getPrepareForHearing());
         assertEquals("http://localhost:8086/", underTest.getUrl());
     }
 }
```

Comment:
Do we require having a space at the end of the file?

Reply:
I usually add it, others usually don't. No strict rules about it.

Reply:
👍